### PR TITLE
added commands to cheatsheet.md

### PR DESCRIPTION
**Pull request by berry2012**
Added some useful commands to cheatsheet.md for example
- How to view kubeconfig output in case it is redacted
- View cert data of a user in kubeconfig
- Get context names

### CODE REVIEW

File: content/en/docs/reference/kubectl/quick-reference.md [hunk 1]
```diff
@@ -71,12 +71,19 @@ KUBECONFIG=~/.kube/config:~/.kube/kubconfig2
 
 kubectl config view
 
+# Show merged kubeconfig settings and raw certificate data and exposed secrets
```
Comment (Ritikaa96) [inline content/en/docs/reference/kubectl/quick-reference.md:74]:
```suggestion
# Show merged kubeconfig settings with raw certificate data and exposed secrets
```

File: content/en/docs/reference/kubectl/quick-reference.md [hunk 2]
```diff
@@ -71,12 +71,19 @@ KUBECONFIG=~/.kube/config:~/.kube/kubconfig2
...
+kubectl config view --raw 
+
 # get the password for the e2e user
 kubectl config view -o jsonpath='{.users[?(@.name == "e2e")].user.password}'
 
+# get the certificate for the e2e user
+kubectl config view --raw -ojsonpath="{.users[?(.name == 'e2e')].user.client-certificate-data}" | base64 -d
```
Comment (Ritikaa96) [inline content/en/docs/reference/kubectl/quick-reference.md:81]:
```suggestion
kubectl config view --raw -ojsonpath='{.users[?(.name == 'e2e')].user.client-certificate-data}' | base64 -d
```

Comment (Ritikaa96) [reply content/en/docs/reference/kubectl/quick-reference.md:81]: single quote worked for me but not the double quote version. is the double quote correct way?

**Review by Ritikaa96 — CHANGES_REQUESTED** [review]

File: content/en/docs/reference/kubectl/quick-reference.md (hunk 2 above)
Comment (berry2012) [reply content/en/docs/reference/kubectl/quick-reference.md:81]: it works as well

**Review by berry2012 — COMMENTED** [review]
added the single quote feedback

File: content/en/docs/reference/kubectl/quick-reference.md [hunk 3]
```diff
@@ -71,12 +71,19 @@ KUBECONFIG=~/.kube/config:~/.kube/kubconfig2
...
 kubectl config view
 
+# Show merged kubeconfig settings and raw certificate data and exposed secrets
+kubectl config view --raw 
+
 # get the password for the e2e user
 kubectl config view -o jsonpath='{.users[?(@.name == "e2e")].user.password}'
 
+# get the certificate for the e2e user
+kubectl config view --raw -ojsonpath='{.users[?(.name == 'e2e')].user.client-certificate-data}' | base64 -d
```
Comment (niranjandarshann) [inline content/en/docs/reference/kubectl/quick-reference.md:81]: kubectl config view --raw -o jsonpath='{.users[?(.name == 'e2e')].user.client-certificate-data}' | base64 -d

space was missing here

**Comment by niranjandarshann** [comment]
@berry2012  Please do a useful changes as guided above. in the command.

**Comment by niranjandarshann** [comment]
looks good to me 
/lgtm

**Review by dipesh-rawat — COMMENTED** [review]
Thanks!

/approve

**Pull request merged**